packages/app/src/navigation/SidebarNavigationRow.cy.tsx

```diff
@@ -1,20 +1,22 @@
 import SidebarNavigationRow from './SidebarNavigationRow.vue'
-import BookIcon from '~icons/cy/book_x24'
-import SettingsIcon from '~icons/cy/settings_x24'
-import TestResultsIcon from '~icons/cy/test-results_x24'
-import PlaceholderIcon from '~icons/cy/placeholder_x24'
```

Comment:
Are we able to delete these svgs in the repo?

Reply:
I meant bloat in time spent by me.
It would take long to manually find where these icons were used and check that they are not anymore.
Deleting icons could conflict with someone else's PR. They could be using the icons I deleted.
Since deleting icons is a "risky action," that would require me to ask everyone to check their use. 
I prefer to do it only once to only block everyone once. 

Sorry I was not clearer in the first comment.

Reply:
The chance of the icons being used in another PR is small though.

I was not planning on deleting these icons just yet, but, if you prefer, I will check and delete them.

Reply:
`test-results_x24.svg` is still used in the `EmptyGenerator.vue` component, I removed the 3 other files.